src/cli.js

```diff
@@ -34,6 +34,7 @@ async function main (opts = {}) {
     inventory,
     autoinstall:  args.autoinstall,
     verbose:      args.verbose,
+    pkgManager:   args.pkgManager?.trim()
```

Comment:
Minor nit: trying to avoid camel-casing args, but what if instead we just did `--yarn` + `--pnpm`?